### PR TITLE
robot can handle GetPIDGains packet and send values back

### PR DESCRIPTION
**Pull request by ejcmd**
The robot should be able to send its PID values if it receives a GetPIDGains packet.
This PR relies on [the PR in REM](https://github.com/RoboTeamTwente/roboteam_embedded_messages/pull/9)

### CODE REVIEW

File: Core/Src/Control/stateControl.c [hunk 1]
```diff
@@ -82,6 +82,9 @@ void stateControl_ResetAngleI(){
 	stateK[body_w].I = 0;
 }
 
+PIDvariables* stateControl_GetPIDValues(){
+	return stateK;
```
Comment (emielsteerneman) [inline Core/Src/Control/stateControl.c:86]: Variable leaving file scope. I suggest doing it like [this](https://github.com/RoboTeamTwente/roboteam_microcontroller4.0/blob/4516897d9af3a64fe50f52cefcb82164405deb86/Core/Src/wheels.c#L200)

File: Core/Src/robot.c [hunk 2]
```diff
@@ -350,6 +354,26 @@ void loop(void){
 		robotStateInfo.wheelSpeed4 = stateInfo.wheelSpeeds[3];
 	}
 
+	if(flagHandlePIDGainValues){
+		robotPIDGains.header = PACKET_TYPE_REM_P_I_D_GAINS;
+		robotPIDGains.remVersion = LOCAL_REM_VERSION;
+		robotPIDGains.id = ID;
+		PIDvariables* stateK = stateControl_GetPIDValues();
+		robotPIDGains.PbodyX = stateK[0].kP;
+		robotPIDGains.IbodyX = stateK[0].kI;
```
Comment (skaeringur97) [inline Core/Src/robot.c]: Shouldn't these be in the form: stateK[body_x].kP ?

File: Core/Src/wheels.c [hunk 3]
```diff
@@ -252,7 +252,9 @@ void wheels_Unbrake(){
 	wheels_braking = false;
 }
 
-
+PIDvariables* wheels_GetPIDValues(){
+	return wheelsK;
```
Comment (emielsteerneman) [inline Core/Src/wheels.c:256]: variable leaving file scope

**Review by emielsteerneman — CHANGES_REQUESTED** [review]

**Comment by emielsteerneman** [comment]
Implemented transmission of packets in the development branch. Just needs the right PID values instead of the placeholder values

**Pull request closed without merging**